client/src/js/components/bhEmployeeFinder.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * The employee finder component
+ *
+ * @module components/bhEmployeeFinder
+ *
+ * @description a component to deal with employee, it let you choose a employee and return back is account_id
+ * 
+ **/
+ 
+/**An employee finder component controller**/
+ 
+function employeeFinderController (employeeService, Appcache){ 	
+ 	var ctrl = this;
+ 	var cache = new Appcache('employeeFinderComponent'); //using appcache to persist the option choice
+
+ 	//search options array
+
+ 	ctrl.searchOptions = [
```

Comment:
https://github.com/IMA-WorldHealth/bhima-2.X/pull/131#discussion_r53782135 has not been fixed yet.  Should we have a discussion as a team about it?